src/core/ext/xds/xds_client_grpc.cc

```diff
@@ -164,13 +167,14 @@ absl::StatusOr<RefCountedPtr<GrpcXdsClient>> GrpcXdsClient::GetOrCreate(
         GRPC_ARG_TEST_ONLY_DO_NOT_USE_IN_PROD_XDS_CLIENT_CHANNEL_ARGS);
     auto channel_args = ChannelArgs::FromC(xds_channel_args);
     return MakeRefCounted<GrpcXdsClient>(
-        std::move(*bootstrap), channel_args,
+        std::string(key), std::move(*bootstrap), channel_args,
```

Comment:
No need to convert to `std::string` here, now that we're passing the parameter to the ctor as `absl::string_view`.